README-development.rst

```diff
@@ -103,4 +104,9 @@ The SDK is packaged as a wheel. In order to generate the wheel you can run:
 
     python setup.py sdist bdist_wheel
 
-This wheel can then be installed via `pip`.
\ No newline at end of file
+This wheel can then be installed via `pip`.
+
+
+
+.. _config template:
```

Comment:
Hey @sunstarer , please link to the github URL for the file here instead of a relative path.